build.sbt

```diff
@@ -120,12 +120,24 @@ lazy val spark = Project("spark", file("spark")).
   dependsOn(util, raster).
   settings(commonSettings: _*)
 
+lazy val s3 = Project("s3", file("backends/s3")).
+  dependsOn(spark).
```

Comment:
I wonder if dependency should be % "provided", like the test kits. Not sure on all the implications. I suppose most directly it would allow us to use different version of geotrellis-s3 library with geotrellis-spark without class path conflicts.